src/components/JsonInput.tsx

```diff
@@ -32,15 +32,15 @@ export const JsonInput: React.FunctionComponent<IFormtronControl> = ({
     >
       {({ value, onChange, nonDraftValue }) => {
         return (
-          <Flex width="100%" alignItems="center">
-            <Box flex="1">
+          <Box>
```

Comment:
`Box` => `Fragment`

Reply:
Component defining inside of return function although valid, is a pattern we should avoid. You should make DraftValue and ThrottleValue HOCs that accept some args.

So then you can have something like
```
const JsonInputBase = ()=> {...}
export const JsonInpu = DraftValue(JsonInputBase, {value: JSON.stringify(value, null, 2)})
```

https://reactjs.org/docs/higher-order-components.html

Reply:
I was kinda modeling DraftValue and ThrottleValue after the React Context API.

Frankly I this is a perfect use case for a Hook. But Hooks didn't exist when I wrote this code lol.